x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/index.tsx

```diff
@@ -44,23 +44,9 @@ export const AllCases = React.memo(() => {
     ({ page, sort }: EuiBasicTableOnChange) => {
       let newQueryParams = queryParams;
       if (sort) {
-        let newSort;
-        switch (sort.field) {
-          case 'state':
-            newSort = SortFieldCase.state;
-            break;
-          case 'created_at':
-            newSort = SortFieldCase.createdAt;
-            break;
-          case 'updated_at':
-            newSort = SortFieldCase.updatedAt;
-            break;
-          default:
-            newSort = SortFieldCase.createdAt;
-        }
         newQueryParams = {
           ...newQueryParams,
-          sortField: newSort,
+          sortField: sort.field as SortFieldCase,
```

Comment:
To avoid the cast
```
const getSortField = (field: string): SortFieldCase => {
  if (field === SortFieldCase.createdAt) {
    return SortFieldCase.createdAt;
  } else if (field === SortFieldCase.state) {
    return SortFieldCase.state;
  } else if (field === SortFieldCase.updatedAt) {
    return SortFieldCase.updatedAt;
  }
  return SortFieldCase.createdAt;
};
```